### PR TITLE
checking news and description preparing for new release [skip vbump]

### PR DESCRIPTION
**Pull request by shajoezhu**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
-## formatters 0.5.4.9008
+## formatters 0.5.5
  * Applied `styler` and resolved package lint. Changed default indentation from 4 spaces to 2.
- * Fixed wrapping and section dividers error.
- * Allowed section divider between header and table body.
  * Added the possibility of setting a general default using `set_default_hsep()` that sets up the option `getOption("formatters_default_hsep")`.
+ * Allowed section divider between header and table body.
+ * Support for combining duplicate referential footnotes.
```
Comment (edelarua) [inline NEWS.md:5]:
```suggestion
 * Added support for combining duplicate referential footnotes.
```

**Review by edelarua — CHANGES_REQUESTED** [review]
@shajoezhu just one minor suggestion!

**Review by edelarua — APPROVED** [review]
Lgtm! Thanks @shajoezhu!!

**Comment by shajoezhu** [comment]
windows check report

```
* using log directory 'd:/RCompile/CRANguest/R-release/formatters.Rcheck'
* using R version 4.3.2 (2023-10-31 ucrt)
* using platform: x86_64-w64-mingw32 (64-bit)
* R was compiled by
    gcc.exe (GCC) 12.3.0
    GNU Fortran (GCC) 12.3.0
* running under: Windows Server 2022 x64 (build 20348)
* using session charset: UTF-8
* checking for file 'formatters/DESCRIPTION' ... OK
* this is package 'formatters' version '0.5.5'
* package encoding: UTF-8
* checking CRAN incoming feasibility ... [11s] Note_to_CRAN_maintainers
Maintainer: 'Joe Zhu <joe.zhu@roche.com>'
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking serialization versions ... OK
* checking whether package 'formatters' can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking for future file timestamps ... OK
* checking 'build' directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... NOTE
Non-standard file/directory found at top level:
  'pkgdown'
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... [1s] OK
* checking whether the package can be loaded with stated dependencies ... [1s] OK
* checking whether the package can be unloaded cleanly ... [1s] OK
* checking whether the namespace can be loaded with stated dependencies ... [0s] OK
* checking whether the namespace can be unloaded cleanly ... [1s] OK
* checking loading without being on the library search path ... [1s] OK
* checking use of S3 registration ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... [9s] OK
* checking Rd files ... [1s] OK
* checking Rd metadata ... OK
* checking Rd line widths ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of 'data' directory ... OK
* checking data for non-ASCII characters ... [1s] OK
* checking LazyData ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking installed files from 'inst/doc' ... OK
* checking files in 'vignettes' ... OK
* checking examples ... [5s] OK
* checking for unstated dependencies in 'tests' ... OK
* checking tests ... [14s] OK
  Running 'testthat.R' [14s]
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in 'inst/doc' ... OK
* checking re-building of vignette outputs ... [2s] OK
* checking PDF version of manual ... [12s] OK
* checking HTML version of manual ... [10s] OK
* checking for detritus in the temp directory ... OK
* DONE
Status: 1 NOTE
```

**Pull request merged**